sway-ir/src/instruction.rs

```diff
@@ -137,8 +137,9 @@ pub enum FuelVmInstruction {
     ReadRegister(Register),
     /// Revert VM execution.
     Revert(Value),
-    /// - Sends a message to an output via the `smo` FuelVM instruction. The first operand must be
-    /// a `B256` representing the recipient. The second operand is the message data being sent.
+    /// - Sends a message to an output via the `smo` FuelVM instruction.
```

Comment:
`-` will consistently render each point as a bullet point. Original doc rendering started with a bullet point and ended up in a quite messy rendering because it was not clear where the next bullet point is. That doc could, IMO, be in free text without bullet points, but since original author went with bullet points, I decided to respect that and just fix the rendering.